Formula/s/screenpipe.rb

```diff
@@ -37,14 +38,8 @@ def install
   end
 
   test do
-    assert_match "Usage", shell_output("#{bin}/screenpipe -h")
-
-    log_file = testpath/".screenpipe/screenpipe.#{Time.now.strftime("%Y-%m-%d")}.log"
-    pid = spawn bin/"screenpipe --debug setup"
-    sleep 200
-
-    assert_path_exists log_file
-    assert_match "screenpipe setup complete", File.read(log_file)
+    version_output = shell_output("#{bin}/screenpipe -V").strip
+    assert_match /^screenpipe \d+\.\d+\.\d+$/, version_output
```

Comment:
we need a substantial function test

Reply:
i mean the main feature of screenpipe is screen and audio recording, which is not something available in CI